linera-client/src/util.rs

```diff
@@ -17,6 +21,16 @@ pub fn parse_millis_delta(s: &str) -> Result<TimeDelta, ParseIntError> {
     Ok(TimeDelta::from_millis(s.parse()?))
 }
 
+pub fn parse_chain_set(s: &str) -> Result<HashSet<ChainId>, CryptoError> {
+    match s.trim() {
+        "" => Ok(HashSet::new()),
+        s => s
+            .split(",")
+            .map(|s| ChainId::from_str(s))
+            .collect::<std::result::Result<HashSet<_>, _>>(),
```

Comment:
```suggestion
            .map(ChainId::from_str)
            .collect::<Result<HashSet<_>, _>>(),
```

Actually, it should even work without annotation?

```suggestion
            .map(ChainId::from_str)
            .collect(),
```

Reply:
Thanks!